src/components/Table/PaginationTable.tsx

```diff
@@ -1,197 +1,56 @@
-            driver: 'Sheryl Nielsen',
-            country: 'Greece',
-            mileage: '365784',
-          },
-          {
-            truck: 'G-series',
-            driver: 'Benton Gomez',
-            country: 'France',
-            mileage: '80957',
-          },
-        ]);
-      }
-    }
-  };
+  }, []);
 
   return (
-    <>
-      {customPagination && (
-        <tds-table
-          ref={paginationTable}
-          vertical-dividers="false"
-          compact-design="false"
-          enable-responsive
-          no-min-width
-        >
-          <tds-table-header>
-            <tds-header-cell column-key="truck" column-title="Truck type"></tds-header-cell>
-            <tds-header-cell column-key="driver" column-title="Driver name"></tds-header-cell>
-            <tds-header-cell column-key="country" column-title="Country"></tds-header-cell>
-            <tds-header-cell
-              column-key="mileage"
-              column-title="Mileage"
-              text-align="right"
-            ></tds-header-cell>
-          </tds-table-header>
-          <tds-table-body body-data={JSON.stringify(data)}></tds-table-body>
-          <tds-table-footer ref={tableFooter} pagination pages={2}></tds-table-footer>
-        </tds-table>
-      )}
-      {!customPagination && (
-        <tds-table vertical-dividers="false" compact-design="false" enable-responsive no-min-width>
-          <tds-table-header>
-            <tds-header-cell column-key="truck" column-title="Truck type"></tds-header-cell>
-            <tds-header-cell column-key="driver" column-title="Driver name"></tds-header-cell>
-            <tds-header-cell column-key="country" column-title="Country"></tds-header-cell>
-            <tds-header-cell
-              column-key="mileage"
-              column-title="Mileage"
-              text-align="right"
-            ></tds-header-cell>
-          </tds-table-header>
-          <tds-table-body ref={paginationTableBody}></tds-table-body>
-          <tds-table-footer pagination rows-per-page="2"></tds-table-footer>
-        </tds-table>
-      )}
-    </>
+    <tds-table ref={paginationTable} responsive no-min-width>
+      <tds-table-header>
+        <tds-header-cell column-key="truck" column-title="Truck type"></tds-header-cell>
+        <tds-header-cell column-key="driver" column-title="Driver name"></tds-header-cell>
+        <tds-header-cell column-key="country" column-title="Country"></tds-header-cell>
+        <tds-header-cell
+          column-key="mileage"
+          column-title="Mileage"
+          text-align="right"
+        ></tds-header-cell>
+      </tds-table-header>
+      <tds-table-body>
+        {data.map((object, index) => (
+          <tds-table-body-row key={index}>
+            <tds-body-cell>{object.truck}</tds-body-cell>
+            <tds-body-cell>{object.driver}</tds-body-cell>
+            <tds-body-cell>{object.country}</tds-body-cell>
+            <tds-body-cell>{object.mileage}</tds-body-cell>
```

Comment:
The mileage Table column title seems to be right-aligned. Also you can't hover the columns in the pagination Table, only the title (rather than the whole column) gets the hover effect
![image](https://github.com/scania-digital-design-system/tegel-react-demo/assets/48509966/a611458d-2638-451f-8ef1-c352bb67d9e0)